backend/build.gradle.kts

```diff
@@ -135,4 +139,19 @@ fun setUpOpenApiGenerator() {
 
 dependencyCheck {
     failBuildOnCVSS = 0.0f
+    suppressionFile = "cve-suppress.xml"
+}
+
+dockerCompose {
+    useComposeFiles.set(
+        listOf("../docker-compose.yml")
+    )
+    isRequiredBy(tasks.test)
+}
+
+flyway {
+    url = "jdbc:postgresql://localhost:5432/postgres"
+    user = "postgres"
+    password = "postgres"
```

Comment:
Is this a real password? Or production password in plaintext in the code?

Reply:
This is just a password, not a real one and definitely not a production one, but I understand your concern :) I just filled it here for the sake of readability

Reply:
Ok then. Just take it from system env or something similar and I am happy :).

Reply:
We're not really running this plugin, this plugin is here only for the future purpose if we might need to repair flyway migrations (checksums etc) or to run some script without running the app, so I don't really see a point of using system env here